shell/platform/windows/client_wrapper/flutter_view_controller.cc

```diff
@@ -0,0 +1,65 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "include/flutter/flutter_view_controller.h"
+
+#include <algorithm>
+#include <iostream>
+
+namespace flutter {
+
+FlutterViewController::FlutterViewController(
+    const std::string& icu_data_path)
+    : icu_data_path_(icu_data_path) {
```

Comment:
Longer term, we'll want to remove this argument and the engine-related arguments in `CreateFlutterView`, and instead pass in an instance of an object that represents an engine (à la [`FlutterEngine`](https://github.com/flutter/engine/blob/e22893966a24c2a3facf133c158437b40533297d/shell/platform/darwin/ios/framework/Headers/FlutterEngine.h))

Reply:
OK.  Is this actionable now?

Reply:
Please reopen if there is anything actionable.